src/components/Add-items/Add-items.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const AddItems = () => {
+  return (
+    <div className="container-main">
```

Comment:
Improvements:

It's a good practice for many reasons to use HTML Semantic. Therefore, I might suggest using the [`<main>` tag](https://developer.mozilla.org/es/docs/Web/HTML/Element/main) versus class name `container-main`. 😄